x-pack/plugins/ml/server/models/data_recognizer/modules/apm_transaction/manifest.json

```diff
@@ -1,29 +1,29 @@
 {
   "id": "apm_transaction",
   "title": "APM",
-  "description": "Detect anomalies in high mean of transaction duration (ECS).",
+  "description": "Detect anomalies in transactions from your APM services",
   "type": "Transaction data",
   "logoFile": "logo.json",
-  "defaultIndexPattern": "apm-*",
+  "defaultIndexPattern": "apm-*-transaction",
```

Comment:
Just checking that this only has the `*` wildcard in the middle of the pattern, and not at the end too, as we have `apm-*-transaction-*` in the example provided to the setup endpoint.

Reply:
there are two index patterns in the test data provided by @dgieselaar : `apm-*` and `apm-*-transaction`. I went with the latter since the module `apm_transaction` focuses only on transaction data. @dgieselaar will `apm-*-transaction` reliably exist? If not, @peteharverson are there any potential consequences of having a nonexistent index pattern here?

Reply:
@blaklaybul it's configurable. I'm assuming we set this when we create the job (have to check but can't find the code right now).

Reply:
@blaklaybul the `defaultIndexPattern` supplied in the module `manifest.json` is just used as a fallback by our module endpoints if no `indexPatternName` is supplied to the endpoint. So for the common use case, where the index pattern is supplied to the `setup` endpoint to create the jobs, this value from the manifest won't be used.

Reply:
thanks @peteharverson